assets/components/openshift-router/deployment.yaml

```diff
@@ -87,12 +87,15 @@ spec:
             - name: http
               containerPort: 80
               protocol: TCP
+              hostPort: 80
             - name: https
               containerPort: 443
               protocol: TCP
+              hostPort: 443
             - name: metrics
               containerPort: 1936
               protocol: TCP
+              hostPort: 1936
```

Comment:
We should probably not export the metrics on the host.

Reply:
Ack, removed.